client/executor/common/src/wasm_runtime.rs

```diff
@@ -42,7 +42,11 @@ pub enum InvokeMethod<'a> {
 		///
 		/// Function pointer, index into runtime exported table.
 		dispatcher_ref: u32,
-		/// Actual wasm function pointer (index into table) that should be invoked.
+		/// Extra argument for dispatch.
+		///
+		/// Common usage would be to use it as an actual wasm function pointer
+		/// that should be invoked, but can be used as any extra argument on the
+		/// callee side.
```

Comment:
👍

Reply:
fwiw, I am not entirely sure that those are contradicting:

dispatcher would pass control to some "function" ultimately and I don't think it really matters how that function is dispatched,: a function call into a table or thru a switch, or if the functions are compiled in and from the outside the dispatcher ends up not calling anything.